ddtrace/tracing/trace_handlers.py

```diff
@@ -0,0 +1,204 @@
+import functools
```

Comment:
Should we rename this module to `wsgi_trace_handlers.py`? Do we plan to use this module to support tracing for all integrations?

Reply:
In future changes I plan to separate these handlers into logical units each with its own file. At the moment I don't know exactly what those units will be - hopefully not one per contrib!